torch/_dynamo/trace_rules.py

```diff
@@ -132,6 +134,32 @@
     "torch.resize_as_": SkipFilesVariable,
     "torch.resize_as_sparse_": SkipFilesVariable,
     "torch.get_default_device": TorchInGraphFunctionVariable,
+    # functorch
+    "torch._functorch.vmap._check_int_or_none": UserFunctionVariable,
```

Comment:
In a follow-up PR we should ask @yanboliang if the skipfiles refactor has landed and if we can clean these up